rider-core/src/main/java/com/github/database/rider/core/util/DateUtils.java

```diff
@@ -0,0 +1,19 @@
+package com.github.database.rider.core.util;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+public class DateUtils {
+
+    private static final SimpleDateFormat DBUNIT_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
```

Comment:
Is it necessary to have a single instance of `SimpleDateFormat`? You could construct the object locally to avoid this synchronization code.

Reply:
Agreed, most of the time our code will not be running in parallel so it doesn't make sense the synchronization...I was concerned on reuse the date format, now I'm just reusing the string describing the format, see changes here: 387ba0d6881572032dacc1ffaec3e7c67f2e0955

Thanks